funnel/views/event.py

```diff
@@ -209,4 +209,6 @@ def event_checkin(profile, space, event, participant):
     a.checked_in = checked_in
     db.session.add(a)
     db.session.commit()
+    if request.method == 'POST':
```

Comment:
There's actually two different things happening here.

1. `request.xhr` implies (only) that the request came from JS and so a fragment response may be appropriate. It does not indicate whether the response should be a HTML fragment or JSON.
2. The HTTP `Accepts` header is a list of content types that the client is expecting. Browsers send a full list starting with 'text/html' and ending with '*/*'. You can indicate that you'd like a JSON response by sending 'application/json'.

Coaster provides a [`render_with` decorator](https://github.com/hasgeek/coaster/blob/master/coaster/views.py#L388) that will make this easier to write. I'll recommend using that.

@karthikb351 for Android, since client installations have life in the wild that is out of our control, all APIs called from the Android app should be version controlled and maintained independently of the web APIs. We'll need to track usage of those APIs independently to know when it is safe to retire them.